grpc/src/main/java/io/stargate/grpc/codec/cql/ValueCodec.java

```diff
@@ -15,13 +15,16 @@
  */
 package io.stargate.grpc.codec.cql;
 
+import com.datastax.oss.driver.api.core.ProtocolVersion;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import io.stargate.db.schema.Column.ColumnType;
 import io.stargate.proto.QueryOuterClass.Value;
 import java.nio.ByteBuffer;
 
 /** Codec to convert to/from gRPC and CQL native protocol ({@link ByteBuffer}) values. */
 public interface ValueCodec {
+  ProtocolVersion PROTOCOL_VERSION = ProtocolVersion.DEFAULT;
```

Comment:
Should the protocol version be the `Persistence` default rather than the driver's default? Even if we use driver codecs, I guess it might be preferable to use the latest version that `Persistence` supports. WDYT?

Reply:
I agreee, but I tried doing this, but the java-driver's `ProtocolVersion` is an interface and was not straightforward to map from Stargate's protocol version. I'll give it another try.